RubrikPolaris/M365/Start-MassRecovery.ps1

```diff
@@ -75,31 +86,31 @@ function Start-MassRecovery() {
             @{
                 SnappableType = "O365_ONEDRIVE";
                 SubSnappableType = "NONE";
-                NameSuffix="_OneDrive";
+                NameSuffix="OneDrive";
             }
         );
         "Exchange" = @(
             @{
                 SnappableType = "O365_EXCHANGE";
                 SubSnappableType = "O365_MAILBOX";
-                NameSuffix="_Mailbox";
+                NameSuffix="Mailbox";
```

Comment:
it would be good to revert these, as in the UI it is better to suffix with "<given_name>_Mailbox" rather than "<give_name>Mailbox"